index.js

```diff
@@ -88,9 +94,18 @@ export class Placement {
  * @returns Promise
  */
 export function load_placements() {
-  // Find all elements matching required data binding attribute
-  const node_list = document.querySelectorAll("[" + ATTR_PREFIX + "publisher]");
-  const elements = Array.prototype.slice.call(node_list);
+  // Find all elements matching required data binding attribute. We don't yet
+  // support multiple placements on the ad-server. For now, this could result in
+  // competing ad placements.
+  let elements = [];
+  const selector = "[" + ATTR_PREFIX + "publisher]";
+  if (SUPPORTS_MULTIPLE_PLACEMENTS) {
+    const node_list = document.querySelectorAll(selector);
+    elements = Array.prototype.slice.call(node_list);
+  } else {
+    const node = document.querySelector(selector);
```

Comment:
Do we want to log an error if there _are_ more than 1 placement? I could see people using a theme with support and then trying to add something custom and getting confused, for example.

Reply:
Ah yeah, perhaps. That might be a better approach and wouldn't alter the code.